flight-reservation/components/TextInputComponent/index.js

```diff
@@ -1,19 +1,36 @@
-import { AntDesign } from "@expo/vector-icons";
-import React from "react";
+import { Ionicons } from "@expo/vector-icons";
+import React, { useState } from "react";
 import { styles } from "./styles";
 import { Text, TextInput, View } from "react-native";
 
-const TextInputComponent = ({ title, icon, help }) => {
```

Comment:
Dont remove the param icon, should be available for other cases

Reply:
addressed